commit eddc11c2e96a60f44e176833266b4c342d1b7b24 (refs/changes/81/81/3)
Author: Dave Pacheco <dap@joyent.com>
Date:   2016-07-14T20:59:38+00:00 (3 years, 3 months ago)
    
    TOOLS-1485 document use of project owners on cr.joyent.us
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/TODO.md b/TODO.md
index 2bf129f..6ff3334 100644
--- a/TODO.md
+++ b/TODO.md
@@ -1,12 +1,19 @@
+High priority:
 * Try 2.12.2 + patch to fix the login bug people are hitting
 * Add nginx image to this repository
-* Would be nice to have a plugin to auto-update commit message with reviewers
-  and approved-by (without the other metadata that Gerrit likes to add)
-* backup story for github repos?
-* are the "project owner" settings what we want?
-* auto-import people's public keys and email addresses from GitHub?
-* set up commit message validation
+
+RFEs:
+* add "reviewers" plugin for setting per-project default reviewers
+* plugin to automatically add first ticket identifier as a change's topic?
+* plugin for commit message validation
+* plugin to auto-update commit message with reviewers and approved-by
+  (without the other metadata that Gerrit likes to add)
 * set up jenkins integration for 'make check'
 * set up jenkins integration for 'make test'
+* auto-import people's public keys and email addresses from GitHub?
+
+Other:
+* backup story for github repos?
 * any kind of monitoring?
-* move to docker compose or some other way to keep track of the whole app?
+* look at ContainerPilot
+* look at docker compose or some other way to keep track of the whole app
diff --git a/docs/operator/README.md b/docs/operator/README.md
index c76368d..152975e 100644
--- a/docs/operator/README.md
+++ b/docs/operator/README.md
@@ -82,6 +82,14 @@ administrators to create or import projects.  There's a deprecated "Project
 Creator Role" where we experimented with delegating this privilege.
 
 
+#### Project owners
+
+Gerrit supports project owners that have additional access rights on
+repositories.  We do not use these.  The whole team (and potentially community
+members to whom we delegate) have rights to approve and integrate changes on all
+repositories.
+
+
 ## Administrative tasks
 
 ### Setting up new users
